pkgs/tools/compression/flips/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gtk3, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "flips";
+  version = "1.31";
+
+  src = fetchFromGitHub {
+    owner = "Alcaro";
+    repo = "Flips";
+    rev = "24e6c77eb8d1ed6d0eb6da400c67ea229eca2278";
+    sha256 = "17b9mbpj59w6n5nin9g98rz4q9nz42m5wdir53jaw8cglz9xmsyp";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ gtk3 ];
+
+  installPhase = ''
```

Comment:
The upstream `Makefile` has an `install` rule, does it not work? It looks like it should work.

Reply:
It does, but it doesn't exist in 1.31.